src/components/DisplayTodos.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useTheme, useTodo } from "../state/stateIndex";
+import { FaEdit } from "react-icons/fa";
+import { MdDeleteForever } from "react-icons/md";
+import styles from "../styles/DisplayTodos.module.css";
+const DisplayTodos = () => {
+  const { todoState } = useTodo();
+  const { theme } = useTheme();
+  console.log(todoState);
+  return (
+    <div className={`${styles.display_todo_container} `}>
+      {todoState.map((ele, index) => {
+        console.log(ele.title, index);
+        return (
+          <div
+            className={`${styles.individual_todo} ${
+              theme === "dark" ? styles.dark : styles.light
+            }`}
+          >
+            <p>
+              {ele.title.trim().length > 20
```

Comment:
Suggestion - You can extract this logic into a function and return the output to show here.